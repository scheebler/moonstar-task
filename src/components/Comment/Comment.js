import React from 'react';
import CommentContainer from './styled/CommentContainer';
import EmailContainer from './styled/EmailContainer';
import NameContainer from './styled/NameContainer';

const Comment = ({ comment }) => {
	const { name, email, body } = comment;
	return (
		<CommentContainer>
			<NameContainer>{name}</NameContainer>
			<EmailContainer>{email}</EmailContainer>
			<div>{body}</div>
		</CommentContainer>
	);
};

export default Comment;