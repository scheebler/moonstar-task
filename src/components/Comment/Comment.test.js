import React from 'react';
import Comment from './Comment';
import renderer from 'react-test-renderer';

describe('Jest Comment component', () => {
	it('Test Comment', () => {
		const mockedComment = {
			name: 'Mocked Name',
			email: 'mocked@email.com',
			body: 'This is a mocked body',
		};
		const tree = renderer.create(<Comment comment={mockedComment} />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});