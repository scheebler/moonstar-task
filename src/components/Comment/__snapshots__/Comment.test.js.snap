// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Comment component Test Comment 1`] = `
<div
  className="sc-beqWaB bxEIKC"
>
  <div
    className="sc-dmqHEX dVcblq"
  >
    Mocked Name
  </div>
  <div
    className="sc-gueYoa eZgvgL"
  >
    mocked@email.com
  </div>
  <div>
    This is a mocked body
  </div>
</div>
`;
